src/components/CreateVault.tsx

```diff
@@ -107,7 +107,7 @@ const CreateVault = () => {
           Creating New Vault
         </div>
         <button
-          className="transition mr-1 text-[#A3A5B9] uppercase rounded-[6px] border-2 border-solid border-[#A3A5B9] py-2 px-7 leading-[14px] font-semibold text-xs bg-[#A3A5B9] bg-opacity-0 hover:bg-opacity-10 active:bg-opacity-20"
+          className="font-altSans transition mr-1 text-[#A3A5B9] uppercase rounded-[6px] border-2 border-solid border-[#A3A5B9] py-3 px-7 leading-[14px] font-bold text-xs bg-gray-500 bg-opacity-0 hover:bg-opacity-10 active:bg-opacity-20"
```

Comment:
I see this new font applied to four buttons. They all share these features `font-bold font-altSans uppercase` (maybe more). Consider naming this button styling case with a classname.

Reply:
Done